test/unit/specs/utils/validation/createLiteralValidator.spec.js

```diff
@@ -0,0 +1,11 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
```

Comment:
Is this intentionally empty?

Reply:
I had to add the file to pass the check. I wanted to get some feedback on the design so I submitted the PR, but I plan on adding a real test here.